Add plugin: Graphs

DylanHojnoski opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/DylanHojnoski/obsidian-graphs

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [ ]  macOS
  - [x]  Linux
  - [x]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["fundingUrl": "",](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/manifest.json#L9)
If you don't have one omit this.

[interface ObsidianGraphsSettings {](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L8-L14)
Since you don't use settings in your plugin, remove this.
[async loadSettings() {](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L97-L103)

[const codeblocks: HTMLElement[]  = element.findAll("code");](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L25)
Use a codeblock post processor instead: <https://docs.obsidian.md/Plugins/Editor/Markdown+post+processing#Post-process+Markdown+code+blocks>

DylanHojnoski commented:
> ["fundingUrl": "",](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/manifest.json#L9) If you don't have one omit this.
> 
> [interface ObsidianGraphsSettings {](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L8-L14) Since you don't use settings in your plugin, remove this. [async loadSettings() {](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L97-L103)
> 
> [const codeblocks: HTMLElement[] = element.findAll("code");](https://github.com/DylanHojnoski/obsidian-graphs/blob/e883f47644babaa2e33dea26b469338b81d185d1/main.ts#L25) Use a codeblock post processor instead: https://docs.obsidian.md/Plugins/Editor/Markdown+post+processing#Post-process+Markdown+code+blocks

* removed fundingURL
* removed all references to settings
* replaced markdown post processor with codeblock post processor
